pkg/apis/multicluster/v1alpha1/gameserverallocationpolicy.go

```diff
@@ -32,9 +32,14 @@ type GameServerAllocationPolicySpec struct {
 
 // ClusterConnectionInfo defines the connection information for a cluster
 type ClusterConnectionInfo struct {
-	ClusterName         string   `json:"clusterName"`
+	// Optional: the name of the targeted cluster
+	ClusterName string `json:"clusterName"`
+	// The endpoint for the allocator service in the targeted cluster
 	AllocationEndpoints []string `json:"allocationEndpoints"`
-	SecretName          string   `json:"secretName"`
+	// The name of the secret that contains TLS client certificates to connect the allocator server in the targeted cluster
+	SecretName string `json:"secretName"`
+	// the cluster namespace to allocate gameserver from
```

Comment:
nit: start comment with a capital.

Reply:
another nit: I think it's a bit more proper to phrase it as "The cluster namespace from which to allocate gameservers" since ending sentences with 'from' or 'to' leaves the reading thinking you forgot to type the rest of it.

Reply:
Done. Thanks for the suggestion.